test/functional/apps/visualize/_linked_saved_searches.ts

```diff
@@ -1,138 +0,0 @@
-/*
- * SPDX-License-Identifier: Apache-2.0
- *
- * The OpenSearch Contributors require contributions made to
- * this file be licensed under the Apache-2.0 license or a
- * compatible open source license.
- *
- * Any modifications Copyright OpenSearch Contributors. See
- * GitHub history for details.
- */
-
-/*
- * Licensed to Elasticsearch B.V. under one or more contributor
- * license agreements. See the NOTICE file distributed with
- * this work for additional information regarding copyright
- * ownership. Elasticsearch B.V. licenses this file to you under
- * the Apache License, Version 2.0 (the "License"); you may
- * not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied.  See the License for the
- * specific language governing permissions and limitations
- * under the License.
- */
-
-import expect from '@osd/expect';
-import { FtrProviderContext } from '../../ftr_provider_context';
-export default function ({ getService, getPageObjects }: FtrProviderContext) {
-  const browser = getService('browser');
-  const filterBar = getService('filterBar');
-  const retry = getService('retry');
-  const testSubjects = getService('testSubjects');
-  const PageObjects = getPageObjects([
-    'common',
-    'discover',
-    'visualize',
-    'header',
-    'timePicker',
-    'visChart',
-  ]);
-
-  describe('saved search visualizations from visualize app', function describeIndexTests() {
```

Comment:
were these tests also migrated?

Reply:
yeah would like to re-produce the saved search tests. It doesn't have to be table vis. For example, we could use chart vis. First of all, I see there are a lot of detailed concepts been tested and I don't think charts can reproduce all of them. Secondly, we are trying to ship all the tests to cypress. I think I could reproduce more of these tests. Of course, another way to find another vis type, re-design some tests and use this one as a reference.